Runtime/BacktraceClient.cs

```diff
@@ -289,6 +292,10 @@ public static BacktraceClient Initialize(BacktraceConfiguration configuration, D
             var backtrackGameObject = new GameObject(gameObjectName, typeof(BacktraceClient), typeof(BacktraceDatabase));
             BacktraceClient backtraceClient = backtrackGameObject.GetComponent<BacktraceClient>();
             backtraceClient.Configuration = configuration;
+            if (attachments != null && attachments.Count != 0)
```

Comment:
Realize that when getting passed a `List<String>`, the API user can add paths to it later, since it's an object reference. I'd set the reference regardless, even if it's empty.

If the intent of the API is to pass them and never change them (which is fair) you should clone the list (otherwise you're exposing your privates)

Reply:
It's unsupported by native libraries that we're using. To prevent this we're making this list readonly and user won't change it over the time or is aware if he will change the reference to the object just after running some kind of api he can't exect this will change also in our class. 

 We allow user to set attachments on startup - I added a note about it on the diff description